docs/adminguide/hosting/parsec.env

```diff
@@ -3,27 +3,25 @@ PARSEC_HOST=0.0.0.0
 
 # The SSL key file.
 PARSEC_SSL_KEYFILE=/run/secrets/parsec-pem-key
+
 # The SSL certificate file.
 PARSEC_SSL_CERTFILE=/run/secrets/parsec-pem-crt
+
 # Enforce HTTPS by redirecting HTTP request.
 PARSEC_FORWARD_PROTO_ENFORCE_HTTPS=X-Forwarded-Proto:https
 
 # The granularity of Error log outputs.
 PARSEC_LOG_LEVEL=WARNING
+
 # The log formatting to use (`CONSOLE` or `JSON`).
 PARSEC_LOG_FORMAT=CONSOLE
+
 # The log file to write to (default to `stderr`).
 # PARSEC_LOG_FILE
 
-# The URL to reach Parsec server
+# The URL to reach Parsec server.
 PARSEC_BACKEND_ADDR=parsec://127.0.0.1:6777
 
-# Allow organization bootstrap without prior creation.
-PARSEC_SPONTANEOUS_ORGANIZATION_BOOTSTRAP=false
```

Comment:
I think it's still useful to have those variables documented in the template.

Maybe:
```
# Uncomment to allow organization bootstrap without prior creation.
# PARSEC_SPONTANEOUS_ORGANIZATION_BOOTSTRAP=true
```

Reply:
I agree with you. For the moment, I will merge this PR as is and will open another issue to document all variables in this file.

Reply:
Done #6198